src/librustc_trans/trans/cabi_powerpc64le.rs

```diff
@@ -0,0 +1,245 @@
+// Copyright 2014-2016 The Rust Project Developers. See the COPYRIGHT
```

Comment:
This file has a lot of duplicated code from cabi_powerpc64.rs; it needs to be refactored.